spec/unit/installer/analyzer_spec.rb

```diff
@@ -474,6 +509,7 @@ module Pod
             it 'raises if no specs repo with that URL could be added' do
               podfile = Podfile.new do
                 source 'not-a-git-repo'
+                pod 'JSONKit', '1.4'
```

Comment:
Because of the change mentioned above, these Podfiles needed a dependency in them in order to actually update their source.